opium-testing/src/opium_testing.ml

```diff
@@ -0,0 +1,114 @@
+module Testable = struct
+  let status = Alcotest.of_pp Opium_kernel.Status.pp_hum
+  let meth = Alcotest.of_pp Opium_kernel.Method.pp_hum
+  let version = Alcotest.of_pp Opium_kernel.Version.pp_hum
+  let body = Alcotest.of_pp Opium_kernel.Body.pp_hum
```

Comment:
The body won't be tested here if it's a stream. Maybe that's not a problem if the `check_body` function does the job properly.